apps/glusterfs/app.go

```diff
@@ -291,6 +291,11 @@ func (a *App) SetRoutes(router *mux.Router) error {
 			Method:      "POST",
 			Pattern:     "/devices/{id:[A-Fa-f0-9]+}/state",
 			HandlerFunc: a.DeviceSetState},
+		rest.Route{
```

Comment:
@lpabon yeah agreed. so you are suggesting that we can use `/devices/{id:[A-Fa-f0-9]+}` instead of `/devices/{id:[A-Fa-f0-9]+}/remove` with method "POST". Then yes we can do it as we do not require a EntryState.